<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/Resources/caliente.png</normaloff>:/Resources/caliente.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Centigrados</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLCDNumber" name="outCent">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(201, 188, 255);
background-color: rgb(0, 255, 127);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inCent">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="minimum">
          <number>-18</number>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Fahrenheit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="outFar">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(214, 255, 187);
background-color: rgb(170, 170, 255);</string>
         </property>
         <property name="value" stdset="0">
          <double>32.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inFar">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="maximum">
          <number>210</number>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Kelvin</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLCDNumber" name="outKel">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 170, 0);
border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="value" stdset="0">
          <double>273.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inKel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>inCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inFar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outFar</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inKel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outKel</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
